.github/drafts/15938/auditLog-first-draft.md

```diff
@@ -0,0 +1,78 @@
+**Intro**
+
+The audit log is the report of all the activities done in Appsmith. It will automatically capture and display events by recording who performed an activity, what when, and where it was performed.
+
+The main log of events for audits and compliance that we’ll provide is around
+
+
+
+* **Application-specific activities** - when was a change made to a page, who configured a new datasource, who ran a query and when or who deleted an existing app
+* **Administrative configuration changes** - who changed an admin email, when was a new group added or role modified
+* **Sensitive information access** - who last accessed user information or billing information pages
+* **User sign-up and login activity** - when a new user signed up, when was an attempted login unsuccessful and so on
+
+<p style="color: red; font-weight: bold">screenshot or gif of the auditLog screen</p><a href="#gdcalert1">alert1</a>
+
+ 
+
+**Filtering audit logs **
+
+
+
+1. Users - Select the specific user to filter activities based on the user from the dropdown list
+2. Events - Filter events based on events. 
+3. ResourceId - Select the resource-id from the json to filter the 
+4. Date Range - Filter data by specific date range you wish to filter
+
```

Comment:
@jnikhila @Pranay105 not sure who to tag from the user education pod for this but we'll also have to talk about each event and when is it recorded here

We have about 25+ events that we log to start with, here's the list
https://www.notion.so/appsmith/579671b3503b48b1a5967b969e3823e2?v=92a738b1dbc5438c97ca27ca87bc30fd